File: 15_FullStack/Exercise_05.06.24/Exercise_BackEnd/src/main/java/com/example/Exercise_BackEnd/Beans/Meeting.java
package com.example.Exercise_BackEnd.Beans;

import jakarta.persistence.*;
import lombok.*;

@Entity
@NoArgsConstructor
@AllArgsConstructor
@Data
@Table(name="meetings")
@Builder
public class Meeting {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int meetingId;
//    @Column(table = "teams",
//            nullable = false)
    private Integer devId;
    //private DevTeam devTeam;
    private String startTime;
    private String endTime;
    private String description;
    private String roomName;
}

File: 15_FullStack/Exercise_05.06.24/frontend/src/Components/model/meeting.ts
export class Meetings{
    description:string;
    devId:number;
    endTime:string;
    meetingId:number;
    roomName:string;
    startTime:string;

    constructor(description:string,devId:number,endTime:string,meetingId:number,roomName:string,startTime:string){
        this.description=description;
        this.devId=devId;
        this.endTime=endTime;
        this.meetingId=meetingId;
        this.roomName=roomName;
        this.startTime=startTime;
    }
}